openmetadata-docs/content/deployment/docker/index.md

```diff
@@ -4,39 +4,174 @@ slug: /deployment/docker
 ---
 
 # Docker Deployment
+Deploying OpenMetadata in Docker is a great start! 
+Before starting with the deployment make sure you follow all the below Prerequisites.
+## Prerequisites
+### Docker (version 20.10.0 or greater)
+[Docker](https://docs.docker.com/get-started/overview/) is an open-source platform for developing, shipping, and running applications. It enables you to separate your applications from your infrastructure, so you can deliver software quickly using OS-level virtualization. It helps deliver software in packages called Containers.
+
+To check what version of Docker you have, please use the following command.
+```commandline
+docker --version
+```
+
+If you need to install Docker, please visit [Get Docker](https://docs.docker.com/get-docker/).
+### Docker Compose (version v2.1.1 or greater)
+The Docker compose package enables you to define and run multi-container Docker applications. The compose command integrates compose functions into the Docker platform, making them available from the Docker command-line interface ( CLI). The Python packages you will install in the procedure below use compose to deploy OpenMetadata.
+
+- **MacOS X**: Docker on MacOS X ships with compose already available in the Docker CLI.
+- **Linux**: To install compose on Linux systems, please visit the Docker CLI command documentation and follow the
+  instructions.
+
+To verify that the docker compose command is installed and accessible on your system, run the following command.
+
+```commandline
+docker compose version
+```
 
-Deploying OpenMetadata in Docker is a great start! Take a look at our
-[Quickstart](/quick-start/local-deployment) guide to learn how to get OpenMetadata
-up and running locally in less than 7 minutes!
+Upon running this command you should see output similar to the following.
 
-If those steps are already done, you might want to bind Docker Volumes
-for data persistence. Learn how to do so [here](/deployment/docker/volumes).
+```commandline
+Docker Compose version v2.1.1
+```
```

Comment:
There seems to be version of docker compose displayed here inconsistent with the below installation notes. Please keep the versions same in order to avoid any confusions.

Reply:
Done